Build Failing on Windows (x64, Windows 10, VS 2015)

"C:\Haywire\haywire.sln" (default target) (1) ->
"C:\Haywire\haywire_hello_world.vcxproj.metaproj" (default target) (4) ->
"C:\Haywire\haywire_hello_world.vcxproj" (default target) (7) ->
(Link target) ->
  program.obj : error LNK2019: unresolved external symbol opt_config_init refer
enced in function main [C:\Haywire\haywire_hello_world.vcxproj]
  program.obj : error LNK2019: unresolved external symbol opt_config_free refer
enced in function main [C:\Haywire\haywire_hello_world.vcxproj]
  program.obj : error LNK2019: unresolved external symbol opt_args_free referen
ced in function main [C:\Haywire\haywire_hello_world.vcxproj]
  program.obj : error LNK2019: unresolved external symbol opt_config_parse refe
renced in function main [C:\Haywire\haywire_hello_world.vcxproj]
  program.obj : error LNK2019: unresolved external symbol opt_flag_bool referen
ced in function main [C:\Haywire\haywire_hello_world.vcxproj]
  program.obj : error LNK2019: unresolved external symbol opt_flag_int referenc
ed in function main [C:\Haywire\haywire_hello_world.vcxproj]
  program.obj : error LNK2019: unresolved external symbol opt_flag_string refer
enced in function main [C:\Haywire\haywire_hello_world.vcxproj]
  C:\Haywire\builds\windows\debug\haywire_hello_world\haywire_hello_world.exe :
 fatal error LNK1120: 7 unresolved externals [C:\Haywire\haywire_hello_world.vc
xproj]

    134 Warning(s)
    8 Error(s)

It looks like the hello world source set doesn't include opt.c. I don't
have access to any Windows machines to confirm though.
On Mon, 14 Dec 2015 at 20:59, Mike Parsons <notifications@github.com> wrote:

> "C:\Haywire\haywiresln" (default target) (1) ->
> "C:\Haywire\haywire_hello_worldvcxprojmetaproj" (default target) (4) ->
> "C:\Haywire\haywire_hello_worldvcxproj" (default target) (7) ->
> (Link target) ->
> programobj : error LNK2019: unresolved external symbol opt_config_init
> refer
> enced in function main [C:\Haywire\haywire_hello_worldvcxproj]
> programobj : error LNK2019: unresolved external symbol opt_config_free
> refer
> enced in function main [C:\Haywire\haywire_hello_worldvcxproj]
> programobj : error LNK2019: unresolved external symbol opt_args_free
> referen
> ced in function main [C:\Haywire\haywire_hello_worldvcxproj]
> programobj : error LNK2019: unresolved external symbol opt_config_parse
> refe
> renced in function main [C:\Haywire\haywire_hello_worldvcxproj]
> programobj : error LNK2019: unresolved external symbol opt_flag_bool
> referen
> ced in function main [C:\Haywire\haywire_hello_worldvcxproj]
> programobj : error LNK2019: unresolved external symbol opt_flag_int
> referenc
> ed in function main [C:\Haywire\haywire_hello_worldvcxproj]
> programobj : error LNK2019: unresolved external symbol opt_flag_string
> refer
> enced in function main [C:\Haywire\haywire_hello_worldvcxproj]
> C:\Haywire\builds\windows\debug\haywire_hello_world\haywire_hello_worldexe
> :
> fatal error LNK1120: 7 unresolved externals
> [C:\Haywire\haywire_hello_worldvc
> xproj]
>
> 134 Warning(s)
> 8 Error(s)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kellabyte/Haywire/issues/76>.
>

--- a/src/haywire/http_response_cache.c
+++ b/src/haywire/http_response_cache.c
@@ -1,6 +1,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#include <time.h>
 #include "uv.h"
 #include "haywire.h"
 #include "hw_string.h"

